cmd/composeAndExecute.go

```diff
@@ -193,5 +193,5 @@ func composeAndExecute() {
 func init() {
 	rootCmd.AddCommand(composeAndExecuteCmd)
 	composeAndExecuteCmd.Flags().BoolVarP(&recheckHeadersArg, "recheck-headers", "r", false, "whether to re-check headers for watched events")
-	composeAndExecuteCmd.Flags().DurationVarP(&queueRecheckInterval, "queue-recheck-interval", "q", 5 * time.Minute, "how often to recheck queued storage diffs")
+	composeAndExecuteCmd.Flags().DurationVarP(&queueRecheckInterval, "queue-recheck-interval", "q", 5*time.Minute, "how often to recheck queued storage diffs")
```

Comment:
@rmulhol The format/resolution to supply this parameter in is unclear. Seconds?

Reply:
@m0ar duration takes a format like `5m30s` - put some examples in the `flags` section of the [`composeAndExecute` docs](https://github.com/vulcanize/vulcanizedb/blob/staging/documentation/composeAndExecute.md), but happy to augment/move that if you think that'd be helpful.

Reply:
Add it in the short-hand docs here as well? I'd never have guessed :D